editor/gui/editor_run_bar.cpp

```diff
@@ -95,20 +95,30 @@ void EditorRunBar::_notification(int p_what) {
 
 			if (is_movie_maker_enabled()) {
 				main_panel->add_theme_style_override(SceneStringName(panel), get_theme_stylebox(SNAME("LaunchPadMovieMode"), EditorStringName(EditorStyles)));
+
+				write_movie_button->begin_bulk_theme_override();
```

Comment:
You can use a theme variation instead.

Reply:
I haven't done that much, so I'll have to look into how it works. I believe this is how the Movie Maker button styling was done before my PR so at least it's in line with that, but if theme variations would be better for this then I can certainly try to implement them here!